awx/ui/src/components/PaginatedTable/ActionItem.js

```diff
@@ -2,7 +2,13 @@ import 'styled-components/macro';
 import React from 'react';
 import { Tooltip } from '@patternfly/react-core';
 
-export default function ActionItem({ column, tooltip, visible, children }) {
+export default function ActionItem({
+  column,
+  tooltip,
+  visible,
+  position,
```

Comment:
This prop isn't being used anywhere in the code - is this change meant for future-scoped work?

Reply:
@kialam, removed.